src/imageTools/freehand.js

```diff
@@ -631,7 +631,6 @@ function modifyObject (e, nearby) {
   const eventData = e.detail;
```

Comment:
You can probably delete this file if the ported tool covers its use case?

Reply:
Yeah, I just left it in for now for quick personal reference. Now I've had a review I'll delete it (y).